tests/gramex.yaml

```diff
@@ -60,6 +60,15 @@ url:
     kwargs:
         function: nonexistent
 
+  # Test IDE
+  post-get-config:
+    pattern: /ide_config_handler
+    handler: FunctionHandler
+    kwargs:
+      function: gramex.apps.ide.ide.ide_config_handler
```

Comment:
Less is better than more.

- Can we move `gramex.apps.ide.ide.ide_config_handler` to `gramex.apps.ide.ide_config_handler`?

Reply:
This is temporary and would change once IDE code gets integrated. Leaving as is for now